### DISCUSSION TITLE
Build error in resources

### DISCUSSION THREAD

**Discussion by phpiando** [retypeapp/retype#233] [Q&A]
I'm using the "retype build" command to compile the files, however after I run the command and go to the "output" folder of the files I noticed that when I opened the index.html file, the site was completely unconfigured (no css), for some reason it is not finding the .css files in the correct folder..

![image](https://user-images.githubusercontent.com/12565400/144640042-c612fff6-657e-4f7c-9a7d-bec1a66404ff.png)

**Comment by geoffreymcgill** [comment]
I believe this is because you need to open the site using a web server because those resources are emitted with root `/` relative paths. A web server is required. The website should work correctly if the contents of the `output` folder are deployed to a remote web server or run in the context of a local web server, such as running the command `retype run` after you build. 

Or, you can run your own local web server, such as [`static-server`](https://github.com/nbluis/static-server) or [`live-server`](https://npmjs.com/package/live-server). 

Does your website open and function correctly if you run `retype watch` instead of `retype build`?

**Reply by phpiando** [reply]
Yes, using retype watch open normal, That could be it then, I'll use it this way... thanks for the tip
